### PR TITLE
BUG #26: If masking reduces the bounds of a dataset, plots look incom…

### PR DESCRIPTION
**Pull request by danlipsa**
…plete.

We use 'vtk_dataset_bounds_no_mask', the dataset bounds before masking
instead of the dataset bounds for fitToViewport.

### CODE REVIEW

**Comment by danlipsa** [comment]
Data and baselines:
https://github.com/UV-CDAT/uvcdat-testdata/pull/159

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 @sankhesh Please review.

**Comment by danlipsa** [comment]
@doutriaux1 It turned out that what I thought was a bug in VTK was a feture. :smile: So no VTK update is required for this.

File: vcs/vcs2vtk.py [hunk 1]
```diff
@@ -14,6 +14,25 @@
 import sys
 import numbers
 
+_DEBUG_VTK = True
```
Comment (sankhesh) [inline vcs/vcs2vtk.py]: Shouldn't `_DEBUG_VTK` be set to `False`?

Comment (danlipsa) [inline vcs/vcs2vtk.py]: @sankhesh Thanks, I will remove the debug code.

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 Ready to merge?

**Comment by doutriaux1** [comment]
i need to run the tests and look at it, sorry I'm on another project today.

File: vcs/VTKPlots.py [hunk 2]
```diff
@@ -596,8 +598,9 @@ def plot(self, data1, data2, template, gtype, gname, bg, *args, **kargs):
             ren = kargs["renderer"]
 
         vtk_backend_grid = kargs.get("vtk_backend_grid", None)
+        vtk_dataset_bounds_no_mask = kargs.get("vtk_dataset_bounds_no_mask", None)
```
Comment (aashish24) [inline vcs/VTKPlots.py]: None is returned by default but I am okay with more specifics.

**Comment by aashish24** [comment]
@danlipsa looks reasonable to me.

**Comment by aashish24** [comment]
@sankhesh @doutriaux1 can we merge this branch?

**Comment by doutriaux1** [comment]
let me run it, sorry about that.

**Pull request merged**